HW13/task_1.py

```diff
@@ -0,0 +1,120 @@
+import json
+
+
+def stats():
+    count = 0
+    with open("number_book.json") as number_book:
+        file_json_data = number_book.read()
+        file_data = json.loads(file_json_data)
```

Comment:
nit: too many copy/paste code, so it would be better to move to a separate function for reading to file and for writing to file